src/api.js

```diff
@@ -48,6 +48,28 @@ async function getOrgName(id) {
   return null;
 }
 
+/**
+ * Fetches the details of an organization from the steps app database
+ * @param {number} id 
+ */
```

Comment:
Please add a `@returns` tag for this JSDoc seeing as this function returns something.

Reply:
Done